.github/PR_TEMPLATE/1-Basic.md

```diff
@@ -0,0 +1,22 @@
+# Description
+
+Include a summary of the changes and the related issue.
+
+## Link to the issue you are closing
+
+Fixes # (issue)
+
+## Screenshot or other testing that you have completed
+
+Your PR should include at least one test/screenshot in order to showcase that the changes don't break/fix what was needed. In the case of frontend, a screenshot should suffice. For backend, depending on the scenario, some temporary tests may need to be written to show that the function does what's needed.
```

Comment:
Does this mean backend sends a screenshot of the tests passing or should it show I/O of the function?